Add mobile tips JA/EN (#5)
henkaku-center/henkaku-wiki: notes/on_boarding_mobile.md

---

> title: on board tips for mobile (EN)

Reviewer: ```suggestion
title: onboarding tips for mobile (EN)
```

description:
published: true
date: 
tags:
editor: markdown
dateCreated: 
---

> # **HENKAKU on boarding**  
> ## How to join Henkaku Discord Community by iPhone/iPad **Tips**  

Reviewer: ```suggestion
# **HENKAKU onboarding**  
## How to join the Henkaku Discord Community by iPhone/iPad **Tips**  
```

---

> 1. **Creating your own wallet by MetaMask**  
> 	- Open the MetaMask app and create it as instructed.
> 	- Be sure to write down the Secret Recovery Phrase in a secure way. You will need it if you want to use the wallet you have created on another smartphone or PC.
> 	- If you lose your Secret Recovery Phrase, you may not be able to use your wallet.  

Reviewer: ```suggestion
1. **Creating your own wallet with MetaMask**  
	- Open the MetaMask app and create it as instructed.
	- Be sure to write down the Secret Recovery Phrase in a secure way. You will need it if you want to use the wallet you have created on another smartphone or PC.
	- If you lose your Secret Recovery Phrase, you may not be able to use your wallet.  
```

---

> 2. **Fill out the ently form**  
> 	- Enter your first and last name, email, gender, and Ethereum Address (alphanumeric sequence starting with "0x" for wallets created with MetaMask) correctly.  
> 	- Open the wallet and tap on the Ethereum Address to copy it.  
> 	- Enter as specific as possible in Strength, Favorite Project, Your Henkaku, and Contribution.  
> 
> #### About invitations to the community  
> ##### The community is gradually expanding its membership, taking into account the diversity of its members. The community is gradually expanding its membership, taking into account the diversity of its members. Completing the entry form does not mean that you will be invited immediately. Please wait until you receive an invitation to join the community discord from our staff at the email address you entered in the entry form. Please note that we cannot inform you of the timing of the invitation even if you contact us.  

Reviewer: ```suggestion
2. **Fill out the entry form**  
	- Enter your first and last name, email, gender, and Ethereum Address (alphanumeric sequence starting with "0x" for wallets created with MetaMask) correctly.  
	- Open the wallet and tap on your Ethereum Address to copy it.  
	- Enter, in a way as specific as possible, your Strengths, Favorite Project, Your Henkaku, and Contribution.  

#### About invitations to the community  
##### The community is gradually expanding its membership, taking into account the diversity of its members. Completing the entry form does not mean that you will be invited immediately. Please wait until you receive an invitation to join the community discord from our staff at the email address you entered in the entry form. Please note that we cannot inform you of the timing of the invitation even if you contact us.  
```

---

> 3. **How to Join the community**  
> 	- Tap the Discord invite link "https://discord.gg/\..." in the email invitation from the staff. to join the Henkaku community.  
> 	- If you don't have a Discord ID, you'll need to register one here.   
> 	- We recommend that you register your email address when you register your Discord ID. Your login ID will be your registered phone number or email address.  
> 	- Discord recommends that you install the app.  

Reviewer: ```suggestion
3. **How to Join the community**  
	- Tap the Discord invite link `https://discord.gg/\...` on the invitation email from the staff to join the Henkaku community.  
	- If you don't have a Discord ID, you'll need to register one here.   
	- We recommend that you register your email address when you register your Discord ID. Your login ID will be your registered phone number or email address.  
	- Discord recommends that you install the app.  
```

---

> 4. **Initial settings in Discord's Henkaku community**
> 	- **At first, only the channel "🤚｜welcome" is displayed, so tap on that channel. Follow the instructions in the channel and tap the 👍 at the end of the post by "yshimura". If you don't see the 👍 at the end of the post, tap & hold on any part of the post to react to the 👍. If you see a 👍 at the end of the post and it turns translucent purple, you have correctly reacted with a 👍.  
> 	You should now be able to access the other channels.  
>   (If you are prompted to Verify your email address during the process, please follow the instructions.)  
> 	- **Tap on the channel "🚀｜start-here"** and follow the instructions on the channel to complete the process.
> 	  - Tap the following link to the orientation video  
> 		https://vimeo.com/670067622/6b5e3d0f6a  
> 		Tap the "▶️" button in the lower left corner to start the video.  
>     When you're done watching, tap "Done" in the upper left corner to return to Discord.

Reviewer: ```suggestion
4. **Initial settings in the Henkaku Discord community**
	- **At first, only the channel "🤚｜welcome" is displayed, so tap on that channel. Follow the instructions in the channel and tap the 👍 at the end of the post by "yshimura". If you don't see the 👍 at the end of the post, tap & hold on any part of the post to react to the 👍. If you see a 👍 at the end of the post and it turns translucent purple, you have correctly reacted with a 👍.  
	You should now be able to access the other channels.  
  (If you are prompted to Verify your email address during the process, please follow the instructions.)  
	- **Tap on the channel "🚀｜start-here"** and follow the instructions on the channel to complete the process.
	  - Tap the following link to watch the orientation video  
		https://vimeo.com/670067622/6b5e3d0f6a  
		Tap the "▶️" button in the lower left corner to start the video.  
    When you're done watching, tap "Done" in the upper left corner to return to Discord.
```
